src/pages/About.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import {Heading} from '@chakra-ui/react';
 
-const About: React.FC = () => (
-  <div>
-    <h1> About Us Page</h1>
-  </div>
-);
+const About: React.FC = () => {
+  return (
+    <>
```

Comment:
Instead of the empty brackets, use Box (which just defaults to div but I think is cleaner than using <>)